scripts/all.fish

```diff
@@ -0,0 +1,11 @@
+for features in "--all-features" "--features runtime-benchmarks" "--features try-runtime"
```

Comment:
When I try to execute this, I receive an `error: no such subcommand: `workspaces`.

Reply:
You have to install `cargo-workspaces`. I will add a comment for that.